Installer/dolphin-emu.spec

```diff
@@ -56,7 +56,8 @@ BuildRequires:  lzo-devel
 # Disable miniupnpc in OBS for F20
```

Comment:
Why are we carrying this file in our repo vs. letting distro maintainers have their own versioning for it? That would make things a lot easier for everyone.